includes/Core/Admin/Screens.php

```diff
@@ -288,13 +288,16 @@ private function get_screens() {
 						}
 					},
 					'render_callback'  => function( Context $context ) {
+						$setup_slug = $context->input()->filter( INPUT_GET, 'slug', FILTER_SANITIZE_STRING );
+						$reauth = $context->input()->filter( INPUT_GET, 'reAuth', FILTER_SANITIZE_STRING );
```

Comment:
Let's use the same logic for `reAuth` as we had before (and still use elsewhere)
https://github.com/google/site-kit-wp/blob/c26b8c6f62e62e51524592e66464f91d15257e1d/includes/Core/Authentication/Authentication.php#L814
https://github.com/google/site-kit-wp/blob/c26b8c6f62e62e51524592e66464f91d15257e1d/includes/Core/Modules/Module_With_Screen_Trait.php#L55
```suggestion
						$reauth = $context->input()->filter( INPUT_GET, 'reAuth', FILTER_VALIDATE_BOOLEAN );
```

Reply:
https://github.com/google/site-kit-wp/pull/3805/commits/861db7225e67b969e69a3afd7f851016096a7859